pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.black]
+line-length = 80
+
+[tool.isort]
+profile = "black"
+py_version = 36
```

Comment:
We should update this to 38 as the last supported version as per [Status of Python versions](https://devguide.python.org/versions/)

Reply:
That depends on beets. AFAIK beets is still claiming to be backwards compatible to 3.6?

Reply:
We currently claim to support Python 3.7+:
https://github.com/beetbox/beets/blob/7736ae76344ee846e92a912db2cc26478105789a/setup.py#L183

But @arogl is right that we should probably drop abandoned versions of Python (and therefore move to 3.8+ now).

Reply:
Alright, I can update the configuration.